metadata.php

```diff
@@ -45,7 +45,8 @@
     'version'           => '0.1.0',
     'author'            => 'Wirecard',
     'url'               => 'https://www.wirecard.com',
-    'email'             => 'shop-systems-support@wirecard.com',
+    //TODO: change email address after internal QA to: shop-systems-support@wirecard.com
```

Comment:
just to be sure I'll add a change request

Reply:
Passed internal QA, I have changed email back to final one